app/routes/bank/headword-list/$initialLetter.tsx

```diff
@@ -0,0 +1,69 @@
+import type { LoaderArgs } from "@remix-run/node"
+import { Link, useCatch, useLoaderData, useParams } from "@remix-run/react"
+import invariant from "tiny-invariant"
+import { prisma } from "~/db.server"
+
+export async function loader({ params }: LoaderArgs) {
+  invariant(params.initialLetter, "initialLetter not found")
+
+  const query = `${params.initialLetter}%`
+
+  const headwords = await prisma.$queryRaw<
+    {
+      id: number
+      headword: string
+      count: number
+    }[]
+  >`SELECT headword, count('c.id') AS count FROM citation c INNER JOIN headword h ON c.headword_id = h.id INNER JOIN user u ON c.user_id = u.id WHERE UPPER(headword) LIKE UPPER(${query}) GROUP BY UPPER(headword), h.id ORDER BY UPPER(headword)`
+
+  console.log("headwords", headwords)
```

Comment:
also if you're gonna make any changes would probably be nice to remove these logs. Though if you're not making any changes I'll be cleaning these up sometime soon hopefully

Reply:
Good catch. Thanks.